File: publlic_html/CPP/prostokat.cpp
/*
 * prostokat.cpp
 * 
 * Copyright 2018  <>
 */
 


#include <iostream>

using namespace std;

int obwod(int a, int b)
{
   return 2*a + 2*b;
}

int pole(int a, int b)
{
   return a * b;
}
    
int main(int argc, char **argv)
{
      int a, b;  // deklaracja zmiennych
    a = b = 0; //inicjalizacja zmiennych
    
    cout << "Podaj pierwszy bok: ";
    cin >> a;
    
    cout << "Podaj drugi bok: ";
    cin >> b;
    
    cout << a << " " << b;
    
    cout << endl << " Obwód: " << obwod(a, b) << endl;
    cout << endl << " Pole: " << pole(a, b) << endl;
	
	return 0;
}

File: publlic_html/python/prostokat.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  prostokat.py
#  
#  Copyright 2018  <>

def obwod(a, b):
    return a + b + a + b
    
    
def pole(a, b):
    return a * b
    
    

def main(args):
    a = int(input("Podaj pierwszy bok: "))
    print(a)
    b = int(input("Podaj drugi bok: "))
    print(b)
    
    print("Obwód: ", obwod(a, b))
    print("Pole: ", pole(a, b))
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))


